### DISCUSSION TITLE
How to apply flags on Docker instance on Google Cloud Run?

### DISCUSSION THREAD

**Discussion by janswist** [gotenberg/gotenberg#519] [Q&A] [answered]
I found today that my instance of Gotenberg on GCR wasn't running:
![Zrzut ekranu 2022-10-8 o 18 02 46](https://user-images.githubusercontent.com/5583954/194717117-3791b8dc-bcd8-4928-8cca-b301b3e95644.png)
(don't know how, I don't do that many conversions per day and thought GCR closes it down whenever instance finished its job)

So I tried this:
![image](https://user-images.githubusercontent.com/5583954/194717168-da3518f4-ef56-4e22-b192-38f47f2eb69b.png)

...based on this:
![image](https://user-images.githubusercontent.com/5583954/194717077-39863d17-e34d-4bdf-971f-50080b82dac9.png)

But turns out I still get `GET /health` check:
![image](https://user-images.githubusercontent.com/5583954/194717203-29ed8c1d-d35d-41d6-8e14-622235d36ba9.png)
so I reckon my command didn't really worked since health check shouldn't be available.

What am I doing wrong?

P.S. Also any tips to make sure it never happens also appreciated.

**Comment by stumpylog** [comment]
I think being able to apply configuration via the environment would be helpful.  I'm currently unable to configure the command of a Github action service container to match the actual production configuration as it doesn't seem to support changing the container command.

**Comment by gulien** [comment, marked as answer]
@janswist Your command is working as expected. If that was not the case, the health check would have return something like:

```json
{
  "status": "up"
  "details": {
    "uno": {
      "..."
    }
  }
}
```

The health check now only checks if the API is working.

@stumpylog changing a container command is one of the most basic feature of Docker. It should not be a hassle with GitHub action either. I won't come back using environment variable.
